indico/modules/categories/controllers/base.py

```diff
@@ -57,3 +57,13 @@ class RHManageCategoryBase(RHCategoryBase):
     def _checkProtection(self):
         if not self.category.can_manage(session.user):
             raise Forbidden
+
+
+class RHMoveCategoryBase(RHManageCategoryBase):
+    def _checkParams(self):
+        RHManageCategoryBase._checkParams(self)
+        self.destination = self._category_query.filter_by(id=request.values['destination_id'], is_deleted=False).one()
```

Comment:
`Category.get_one(int(request.form['destination_id']), is_deleted=False)` - no need to use `_category_query` here; that's mainly to optimize display pages